src/api/server-routes.ts

```diff
@@ -770,6 +770,11 @@ export function imagesRoutes(): express.Router {
       .catch(err => res.status(200).json({ status: false }));
   });
 
+  router.post('/build-base', (req: express.Request, res: express.Response) => {
+    buildAbstruseBaseImage()
+      .then(() => res.status(200).json({ data: true }));
```

Comment:
I think it would be smart here to `.catch`.

Reply:
done